openedx/features/course_experience/static/course_experience/js/CourseSock.js

```diff
@@ -57,8 +57,8 @@ export class CourseSock {  // eslint-disable-line import/prefer-default-export
 
       // Log open and close events
       const isOpening = $toggleActionButton.hasClass('active');
-      const logMessage = isOpening ? 'User opened the verification sock.'
-          : 'User closed the verification sock.';
+      const logMessage = isOpening ? 'edx.bi.course.sock.toggle_open'
+          : 'edx.bi.course.sock.toggle_close';
       Logger.log(
```

Comment:
Oh. Maybe I misunderstood what was needed. It's only GA we care about from AC, so however that should be accomplished. I would like to understand though so I don't mislead.